portfolio/src/main/webapp/index.html

```diff
@@ -82,6 +88,30 @@ <h2>My Works</h2>
           </li>
       </ul>
       </div>
+      <div id="contact">
+          <h2>Contact Me</h2>
+
+          <p>
+              We can talk about ANYTHING! Feel free to just reach out.
+          </p>
+
+           <form action ="/form-handler" method="POST">
+                <p>Enter your name:</p>
+                <input class = "input" name="name"/>
+                <br/>
+                <p>Enter your email here:</p>
+                <input class = "input" name="email"/>
+                <br/>
+                <p>A fun fact:</p>
+                <textarea id = "message" class = "input" name="fun-fact">ex: Bungee jumping is on my bucket list</textarea>
```

Comment:
```suggestion
                <textarea id="message" class="input" name="fun-fact">
                    ex: Bungee jumping is on my bucket list
                </textarea>
```

If the text gets too long, you can put the text on a new line and indent like in the suggestion above. This helps keeps things neat and easy to read.